src/components/pagination/pagination.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import "./pagination";
+
+const Pagination = ({
+  firstItemOnPage,
+  maxIDBestNewsOnPage,
+  maxIDBestNews,
+  onChangefirstItemOnPage = () => {},
+}) => {
+  const namePrevious = "previous";
+  const nameNext = "next";
+
+  return (
+    <div className="btn-group">
+      <button
+        key="previous"
+        type="button"
+        onClick={() => onChangefirstItemOnPage(namePrevious)}
+        disabled={firstItemOnPage <= 0 ? "disabled" : ""}
+        className="btn btn-info"
+      >
+        Previous
+      </button>
+      <div>
+        {" "}
+        {firstItemOnPage + 1}-{maxIDBestNewsOnPage}{" "}
+      </div>
+      <button
+        key="next"
+        type="button"
+        onClick={() => onChangefirstItemOnPage(nameNext)}
+        disabled={firstItemOnPage + 20 >= maxIDBestNews ? "disabled" : ""}
```

Comment:
Pagination component is responsible for switching pages. But now it knows about `maxIDBestNews` , items count per page (`20`). 
In future, to change `20` items per page to `10` you will have to modify a lot of files. But ideally it should be only one line.
The same issue about `maxIDBestNews` - now `Pagination` component is coupled with items list and can't be reused in other components.

So, when you  create a component it's good to think about few component's things :
- `component purpose` - to change page, in this case
- `component dependencies` 
- `component api` - change page and show current page:

```jsx
const Pagination = ({ onPreviousPage, onNextPage, currentPage, .... }) => 
```